### ISSUE TITLE
Navigation disappears when returning back to search results

### ISSUE THREAD

**Issue by muloka** [Burning-Man-Earth/iBurn-iOS#78]
Steps taken

1. Search for event, e.g. art
2. View full event details
3. Return to event search results

![Image](http://i.imgur.com/KQkLEbI.jpg)

![Image](http://i.imgur.com/tD6nlX1.jpg)

![Image](http://i.imgur.com/cORRhgT.jpg)

![Image](http://i.imgur.com/4UzqmT0.jpg)

**Comment by dfleck** [comment]
Still in version 29 and SUPER annoying.

**Comment by chrisballinger** [comment]
Yeah I know it's shitty, it's been that way for two years. If you know any talented iOS engineers who could help me fix it in the next day or two, let me know. Here's a workaround that usually works:

1. Hit "Search" to hide the keyboard
2. Go to any other tab like Favorites
3. Go back to Events. If it hasn't fixed itself, repeat steps 2 and 3.
